src/weights.jl

```diff
@@ -537,25 +537,22 @@ wmedian(v::RealVector, w::AbstractWeights{<:Real}) = median(v, w)
 
 ###### Weighted quantile #####
 
-# http://stats.stackexchange.com/questions/13169/defining-quantiles-over-a-weighted-sample
-# In the non weighted version, we compute a vector of index h(N, p)
-# and take interpolation between floor and ceil of this index
-# Here there is a supplementary function from index to weighted index k -> Sk
 
 """
     quantile(v, w::AbstractWeights, p)
 
-Compute the weighted quantiles of a vector `x` at a specified set of probability
-values `p`, using weights given by a weight vector `w` (of type `AbstractWeights`).
+Compute the weighted quantiles of a vector ``x`` at a specified set of probability
```

Comment:
Actually, references to Julia objects should use single backticks... :-) And `v` should be changed to `x` in the signature above to match descriptions below (or the other way around).